src/config/dynamic-theme-fixes.config

```diff
@@ -140,6 +140,15 @@ INVERT
 
 ================================
 
+*.cnrtl.fr
+
+CSS
+.tlf_csyntagme {
+    background-color: rgb(22, 89, 0) !important;
```

Comment:
change to:
`background-color: ${rgb(233, 166, 255)} !important;`

This will use (22, 89, 0) when dark theme is used or (233,166, 255) if light theme with dark text.